examples/apps/echo/client/echo_client.py

```diff
@@ -356,15 +356,21 @@ def Main(args=None):
     logger.info("\n Last update to receipt receipt is:\n {}".format(
         json.dumps(receipt_update_retrieve, indent=4)
     ))
-    status = sig_obj.verify_update_receipt_signature(
-        receipt_update_retrieve['result'])
-    if status == SignatureStatus.PASSED:
-        logger.info(
-            "Work order receipt retrieve signature verification Successful")
+    if "result" in receipt_update_retrieve:
```

Comment:
So is this check present in generic client ?

Reply:
yes